content/java/concepts/math-methods/terms/abs/abs.md

```diff
@@ -11,13 +11,13 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `Math.abs()` method returns the absolute value of the argument.
+The **`Math.abs()`** method returns the absolute value of the argument.
 
 An absolute value is the non-negative version of the value, without considering the positive/negative sign.
 
 ## Syntax
 
-```java
+```pseudo
 Math.abs(n)
 ```
 
```

Comment:
I'm also wondering if we should combine the two "Example [x]" section into one? Maybe something like this:

```java
class AbsoluteValue {
  public static void main(String[] args) {
    double x = -6.5;
    int y = 2;
    double z = -340.8;

    System.out.println(Math.abs(x));
    System.out.println(Math.abs(y));
    System.out.println(Math.abs(z));
  }
}
```

And then we could follow up with some text that precedes a \```shell output snippet:

"This will yield the following output:

```shell
6.5
2
340.8
```
"